packages/dominus-settings/settingsCapitals.js

```diff
@@ -29,7 +20,16 @@ _gs.capitals = function(gameId, path) {
       glass: 80
     }
 
+    settings.startingGarrison = {
+      footmen: 24,
+      archers: 24,
+      pikemen: 24,
+      cavalry: 0,
+      catapults: 0
+    }    
+
     settings.incomeInterval = 1000 * 60 * 10;
+    settings.villagePercentageIncome = 0.12;
```

Comment:
Need to fix all places to use _gs instead of _s for villagePercentageIncome.  

https://github.com/dan335/dominus/blob/master/packages/dominus-capitols/client/rp_info_capitol.html#L127

help_capitals.html

Search code for villagePercentageIncome to make sure you get them all.